internal/config/config_util_test.go

```diff
@@ -224,3 +225,31 @@ func Test_ListCacheTtlSecsToDuration_InvalidCall(t *testing.T) {
 func Test_DefaultMaxParallelDownloads(t *testing.T) {
 	assert.GreaterOrEqual(t, DefaultMaxParallelDownloads(), 16)
 }
+
+func TestOverrideWithMaxRetryAttemptsFlag(t *testing.T) {
+	var testCases = []struct {
+		configValue   int
+		flagValue     int
+		isFlagSet     bool
+		expectedValue int
+	}{
+		{34, -1, true, -1},
+		{34, -1, false, 34},
+		{0, 435, true, 435},
+		{0, 0, false, 0},
+		{0, 1, true, 1},
+		{9223372036, -1, false, 9223372036}, // MaxSupportedTtlInSeconds
```

Comment:
Didn't quite get the comment: "MaxSupportedTtlInSeconds". Why should the max supported TTL have any bearing on this flag? Same question for the last subtest.

Reply:
Discussed offline. removed.